grunt-tasks/svg-transform-add-g.js

```diff
@@ -13,11 +13,8 @@ module.exports = function( grunt ) {
 			var fileContent = grunt.file.read( 'svg-min/' + svgFile );
 
 			// Add <g> to each file
-			fileContent = fileContent.slice( 0, fileContent.indexOf('viewBox="0 0 24 24">') + 20 ) +	// opening SVG tag
-						'<g>' +
-						fileContent.slice( fileContent.indexOf('viewBox="0 0 24 24">') + 20, -6 ) + 	// child elements of SVG
-						'</g>' +
-						fileContent.slice( -6 );	// closing SVG tag
+      fileContent = fileContent.replace( /<svg[^>]*>/i, '$&<g>' ); // add <g> after <svg>
```

Comment:
if we run this twice, won't get we `<g><g>`?

although what @gwwar brought up is 2000% relevant here, that non-RegEx-based parsers are quite important here, for a quick implementation we can find only those `<svg` things that aren't followed by `<g` things

```js
const svgWithoutG = [
	// (?:<svg)     - find the opening prefix (non-capture)
	// ([^>]*)      - capture any tag-internal content
	// (?:>(?!<g>)) - followed by a closing `>` which is not also followed by a `<g>`
	/(?:<svg)([^>]*)(?:>(?!<g>))/,
	'<svg\1><g>'
];

const _svgWithoutG = [
	// grab even those with a closing `</g>`
	// then normalize the endings so they all have it
	/(?:(<\/g>)?<\/svg>)/,
	'</g></svg>'
];
```

Note that what this won't do is handle nesting - if we have nested `<g>` groups we'll likely mess them up.

Reply:
> if we run this twice, won't get we <g><g>?

Not a problem: this is always run only on a re-generated file for safety. 
I'd rather keep the simpler form, but nice use of lookahead there, and I didn't know there was a way in JS' runtime to do non-capture. TIL!

Reply:
I meant that when we regenerate the files again the current RegExps will double the group element.

> I'd rather keep the simpler form

sure, but neither one is really complicated… the given ones are just a little misleading because of how they leave out the `<g>` part